packages/backend/src/routers/organizationRouter.ts

```diff
@@ -208,3 +208,27 @@ organizationRouter.post(
         }
     },
 );
+
+organizationRouter.post('/theme', isAuthenticated, async (req, res, next) =>
```

Comment:
I like the theme sub resource on the organisation router! This looks great. Unsure if it should be a `put` since it only updates a single resource? I'm not actually sure about this one